### PR TITLE
Add PythonDockerInterpreter to interpreter group

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
Recently PythonDockerInterpreter added to PythonInterpreter in https://github.com/apache/zeppelin/pull/1654.

### What type of PR is it?
Bug Fix | Improvement

### Screenshots (if appropriate)
- before 
image

- after
image

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: python/src/main/resources/interpreter-setting.json [hunk 1]
```diff
@@ -43,5 +43,17 @@
...
+    "className": "org.apache.zeppelin.python.PythonDockerInterpreter",
+    "properties": {
+    },
+    "editor":{
+      "language": "sh",
+      "editOnDblClick": false
+    }
   }
 ]
+]
```
Comment (Leemoonsoo) [inline python/src/main/resources/interpreter-setting.json]: This is extra one, i think.

**Comment by astroshim** [comment]
@Leemoonsoo It was my mistake. I just fixed. 
Thank you.

**Comment by Leemoonsoo** [comment]
LGTM!

**Comment by Leemoonsoo** [comment]
Merge to master if there're no further comment.

**Pull request closed without merging**